config/src/main/java/com/yahoo/config/subscription/impl/PayloadChecksum.java

```diff
@@ -0,0 +1,48 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.config.subscription.impl;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Checksums of config definition payload or config payload,
+ * md5 and xxhash64 are the supported types at the moment.
+ *
+ * @author hmusum
+ */
+public class PayloadChecksum {
+
+    private static final Pattern hexChecksumPattern = Pattern.compile("[0-9a-zA-Z]+");
```

Comment:
hex ? Perhaps a-fA-F ?

Reply:
I just used the same pattern as before and we use `HexDump.toHexString()`. I'll change the pattern in next PR.